test/unit/test_bulk.py

```diff
@@ -5,7 +5,7 @@
 
 from nose.plugins.attrib import attr
 
-from ..test_base import AssetManagementUnitTest
+from test.test_base import AssetManagementUnitTest
```

Comment:
I run the unit tests in python with nosetests, but this change doesn't hurt anything.